### PR TITLE
Add WUFFI token

### PR DESCRIPTION
**Pull request by wuffi-inu**
Please add WUFFI token

Please make sure you change the original ones `.yaml` fiels in directories `accounts/`, `collections/` or  `jettons/` and do not touch automatically generated .json files in the repository root.

Пожалуйста, убедитесь, что вы изменяете исходные `.yaml` файлы в директориях  `accounts/`, `collections/` или  `jettons/` и не трогаете автоматически сгенерированные файлы `.json` в корне репозитория.

### CODE REVIEW

**Comment by tikto29** [comment]
Where wuffi-inu:patch-1

**Comment by wuffi-inu** [comment]
> Where wuffi-inu:patch-1

Right here: https://github.com/wuffi-inu/ton-assets/tree/patch-1

Do you see it?

**Comment by wuffi-inu** [comment]
@grishkanaruzhka can you help us get this one merged please? What more needs to be done to have one accepted please?

**Comment by tikto29** [comment]
Ok

**Comment by tikto29** [comment]
> @grishkanaruzhka can you help us get this one merged please? What more needs to be done to have one accepted please?

Yes

**Comment by tikto29** [comment]
Ok

**Review by tikto29 — APPROVED** [review]

File: jettons/WUFFI.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Wuffi
+image: "https://wuffi-inu.github.io/assets/WUFFI_Token_Ticker_green_256x256.png"
```
Comment (wuffi-inu) [inline jettons/WUFFI.yaml:2]: @grishkanaruzhka will this work now? Thank you

**Comment by tikto29** [comment]
Approv

**Comment by tikto29** [comment]
https://github.com/tonkeeper/ton-assets/pull/1102#discussion_r1788125349

**Comment by wuffi-inu** [comment]
@grishkanaruzhka please note that we have very good liquidity on this dex: https://dedust.io/pools/EQBtUO6pPxGGB4TBTciXoHiU4HkCZ768dgDFqI6XnOnlaMD8
It would be great if we could get this one merged. Thank you kindly

**Comment by Gene090** [comment]
We can see that the token has already been approved.

**Comment by wuffi-inu** [comment]
> We can see that the token has already been approved.

@Gene090 that's great. Thank you. Is there a way or place for us to check that for future reference please?

**Pull request closed without merging**